siriuspy/siriuspy/devices/lienergy.py

```diff
@@ -0,0 +1,75 @@
+"""."""
+
+from .device import Device as _Device
+
+
+class LIEnergy(_Device):
+    """Linac Energy Measurement Device."""
+
+    class DEVICES:
+        """Devices names."""
+
+        LI = 'LI-Glob:AP-MeasEnergy'
+        ALL = (LI, )
+
+    _properties = (
+        'Dispersion-SP', 'Dispersion-RB',
+        'IntDipole-Mon', 'Energy-Mon', 'Spread-Mon',
+        'MeasureCtrl-Sel', 'MeasureCtrl-Sts')
+
+    def __init__(self, devname):
```

Comment:
since there is only one device, I think this argument should at least be optional.